source/extensions/filters/http/cache/cache_policy.h

```diff
@@ -23,82 +22,96 @@ struct CacheEntryUsability {
    * Value to be put in the Age header for cache responses.
    */
   Seconds age = Seconds::max();
+  /**
+   * Remaining freshness lifetime.
+   */
+  Seconds ttl = Envoy::Seconds::max();
```

Comment:
Drop the `Envoy::`, or also include it on the type and on both things in the instance above.